tests/test_classifier/test_rocauc.py

```diff
@@ -109,6 +109,11 @@ def test_binary_rocauc(self):
 
         # Score the visualizer
         s = visualizer.score(X_test, y_test)
+
+        # Test that score method successfully returns a value between 0 and 1
+        assert 0 <= s <= 1
```

Comment:
A good place to put this test! Is there a similar place for multiclass? Would that be required?

Reply:
Done